services/get_account_details.proto

```diff
@@ -31,195 +43,384 @@ import "duration.proto";
+         * This value SHALL always be a whole number.
          */
         uint64 balance = 7;
 
         /**
-         * If true, no transaction can transfer to this account unless signed by this account's key
+         * A boolean indicating that the account requires a receiver signature
+         * for inbound token transfer transactions.
+         * <p>
+         * If this value is `true` then a transaction to transfer tokens to this
+         * account SHALL NOT succeed unless this account has signed the
+         * transfer transaction.
          */
         bool receiver_sig_required = 8;
 
         /**
-         * The TimeStamp time at which this account is set to expire
+         * The current expiration time for this account.
+         * <p>
+         * This account SHALL be due standard renewal fees when the network
+         * consensus time exceeds this time.<br/>
+         * If rent and expiration are enabled for the network, and automatic
+         * renewal is enabled for this account, renewal fees SHALL be charged
+         * after this time, and, if charged, the expiration time SHALL be
+         * extended for another renewal period.<br/>
+         * This account MAY be expired and removed from state at any point
+         * after this time if not renewed.<br/>
+         * An account holder MAY extend this time by submitting an account
+         * update transaction to modify expiration time, subject to the current
+         * maximum expiration time for the network.
          */
         Timestamp expiration_time = 9;
 
         /**
-         * The duration for expiration time will extend every this many seconds. If there are
-         * insufficient funds, then it extends as long as possible. If it is empty when it expires,
-         * then it is deleted.
+         * A duration to extend this account's expiration.
+         * <p>
+         * The network SHALL extend the account's expiration by this
+         * duration, if funds are available, upon automatic renewal.<br/>
+         * This SHALL NOT apply if the account is already deleted
+         * upon expiration.<br/>
+         * If this is not provided in an allowed range on account creation, the
+         * transaction SHALL fail with INVALID_AUTO_RENEWAL_PERIOD. The default
+         * values for the minimum period and maximum period are currently
+         * 30 days and 90 days, respectively.
          */
         Duration auto_renew_period = 10;
 
         /**
-         * All tokens related to this account
+         * As of `HIP-367`, which enabled unlimited token associations, the
+         * potential scale for this value requires that users consult a mirror
+         * node for this information.<br/>
+         * A list of tokens to which this account is "associated", enabling the
+         * transfer of that token type by this account.
+         * <blockquote>REVIEW NOTE<blockquote>
+         * Should this be deprecated, much as it is in crypto get info?
```

Comment:
Users can get this information here, they just need to know that only the `maxRelsPerInfoQuery` (default 1000) values are returned.

Reply:
I think you should add a note "Only the top `maxRelsPerInfoQuery` (default 1000) token relationships are returned. For the complete list, consult a mirror node."

Reply:
The mirror node part is already two lines above, I've added the reference to the configuration option.